### PR TITLE
Travis

### PR DESCRIPTION
**Pull request by Br3nda**
Added Travis.yml, to use free continuous integration.

Enum was added in python3.4, so only tests in that or later.

### CODE REVIEW

**Comment by Br3nda** [comment]
[Build Status](https://travis-ci.org/Br3nda/pymysensors)

**Comment by MartinHjelmare** [comment]
Nice! @theolind has to decide if Travis testing should be activated for this repo. We could add the travis config anyhow, for fork owners, but best value would be to use it here in the original repo.

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+language: python
+python:
+  - "3.4"
+  - "3.5"
+  - "3.5-dev" # 3.5 development branch
+  - "nightly" # currently points to 3.6-dev
+# command to install dependencies
+install: "pip install -r requirements.txt"
+# command to run tests
+script: nosetests
```
Comment (MartinHjelmare) [inline .travis.yml]: An empty line in the end of the file is missing.

**Comment by Br3nda** [comment]
Dev seems older, only because you've got "version='0.5'," in master and version='0.4' in dev.
I can make a new PR to dev branch.

**Comment by MartinHjelmare** [comment]
Yeah, we haven't been consistent with version bumps and back merging. Best option now is to base your PR off master but direct the PR to dev.

**Comment by MartinHjelmare** [comment]
Closing this one. See #31 instead.

**Pull request closed without merging**